test/api/unit/libs/makeAdmin.test.js

```diff
@@ -0,0 +1,47 @@
+import makeAdmin from '../../../../website/client/src/libs/makeAdmin';
+
+// const item = {
+//   text: 'You are now admin',
+//   success: true,
+// };
+
+// eslint-disable-next-line mocha/no-exclusive-tests
```

Comment:
Can you clean out these comments? Comments that help provide context for the code or test are fine of course, but these look to be leftovers from local test runs.